app/views/releases/_release_status.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if status.success? %>
+  <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
```

Comment:
Could use `icon_tag` helper here as well.

Reply:
If that method was amended to allow passing in additional tags, sure.